### ISSUE TITLE
Mirror index.Rmd into examples.qmd

### ISSUE THREAD

**Issue by coatless** [gadenbuie/countdown#48]
Follow-up to #35, we should port over the logic in the xarginan slides to a Quarto deck:

https://github.com/gadenbuie/countdown/blob/main/docs/index.Rmd

I'll take lead on the syntax and content port. It would be great to get @gadenbuie's 🎨 stylistic touch; but, that's not required.

**Comment by gadenbuie** [comment]
This is a good idea to make sure we have feature parity.

I've also been thinking it would be good to have a big, kitchen sink `test.qmd` with examples of all of the different syntax options and choices. Initially, it could be used for manual testing, but in the long term I might automate testing, e.g. with playright. But just dumping a bunch of examples with a note about expected behavior in a doc would be a really good start.

**Comment by coatless** [comment]
Yes! It would be great to have a single document that goes across the different syntaxes. My only worry is the size limit if `shinylive` components start getting added. Mobile users _definitely_ yell with large file downloads.
